router.go

```diff
@@ -186,7 +186,9 @@ func (r *router) matchAndDispatch(messages <-chan *packets.PublishPacket, order
 						wg.Add(1)
 						go func() {
 							hd(client, m)
-							m.Ack()
+							if client.options.AutoAckDisabled {
```

Comment:
Sorry - I believe this should now be `if !client.options.AutoAckDisabled` (i.e. if AutoAck is not disabled send the ACK). The same applies to the two below). Hopefully this is the last change!

Reply:
Done! Thanks for correcting.